geospaas_harvesting/crawlers.py

```diff
@@ -231,12 +237,42 @@ def _get_links(cls, html):
         return parser.links
 
 
+class PODAACCrawler(WebDirectoryCrawler):
+    """
+    Crawler for harvesting the data of PODAAC
+    """
```

Comment:
Why change the name?
Please keep `OpenDAPCrawler`, this crawler is not exclusively designed for use with the PODAAC repository.

Reply:
`FOLDERS_SUFFIXES` for **PODAACCrawler** and **ThreddsCrawler** are different.
`FILES_SUFFIXES` for **PODAACCrawler** and **ThreddsCrawler** are also different.
However, at the same time, both of them are using OPENDAP with different addressing! So I think it is better to confine that one into PODAAC name. Isn't it?

Reply:
Thredds and OpenDAP are different things:

Thredds is a data repository that uses OpenDAP, among others, to enable access to its data (see https://www.unidata.ucar.edu/software/tds/current/). But it has its own directory structure, which is why the way to explore it is different from "pure OpenDAP" repositories.

The OpenDAPCrawler's purpose is to provide access to "pure OpenDAP" repositories, like Hyrax (https://www.opendap.org/software/hyrax-data-server).
This is used by multiple data providers, not just PODAAC, so I stand by what I wrote earlier.

Reply:
Done in the next commit